src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -220,7 +241,7 @@ boolean sample(String operationName, long traceId, long duration) {
   }
 
   void reportWavefrontGeneratedData(WavefrontSpan span) {
```

Comment:
Should we consider adding timers here and for `reportSpan()` to measure duration/latency added?

Reply:
good idea. `reportSpan` itself is async. Can add it in `reporter.sendSpan` though perhaps we can add this within the tier 1 SDK.